django_dbconn_retry/__init__.py

```diff
@@ -59,7 +59,8 @@ def ensure_connection_with_retries(self: django_db_base.BaseDatabaseWrapper) ->
                         if self._connection_retries >= self.settings_dict.get("DBCONN_RETRY_MAX_RETRIES", 0):
                             _log.error("Reconnecting to the database didn't help %s", str(e))
                             del self._in_connecting
-                            post_reconnect.send(self.__class__, dbwrapper=self)
+                            post_reconnect.send(self.__class__, dbwrapper=self,
+                                                retry_count=dbwrapper._connection_retries)
```

Comment:
I think `dbwrapper` isn't a variable in scope here, this would break hard.. 

(_Also, your IDE would be yelling at you about this line._ 😉)

Let's move line 77 up to 57?

Reply:
Wow nice catch...
That should've been self._connection_retries.

We can't move 77 up because it needs to run after `_connection_retries` has been incremented.

Reply:
@khyang-udemy how about that 2 week pycharm trial we talked about?

Reply:
Rick offered to pair with me to get me going in IntelliJ.
I agreed to it.